Use C99's <stdbool.h> rather than defining our own bool, true, and false.
--- lib/rbmake/rbfile.h.orig	2006-04-22 17:08:58.000000000 -0500
+++ lib/rbmake/rbfile.h	2022-09-23 04:31:22.000000000 -0500
@@ -10,6 +10,7 @@
 #define _RBFILE_H
 
 #include <stdarg.h>
+#include <stdbool.h>
 
 #define RB_FILE_VERSION		2
 
@@ -71,13 +72,6 @@
 typedef int int32;
 #endif
 
-#if !defined(true) && !defined(RB_OMIT_true_AND_false)
-#define true 1
-#define false 0
-#endif
-
-typedef char bool;
-
 /* -------------------- Opaque objects -------------------- */
 
 typedef struct RbFile RbFile;
